package.json

```diff
@@ -39,6 +39,7 @@
   "devDependencies": {
     "fastify": "^3.2.1",
     "fastify-cookie": "^5.0.0",
+    "pre-commit": "^1.2.2",
```

Comment:
`pre-commit` hasn't been maintained for quite some time. I think this should be `husky` instead.

Reply:
oh, we should probably move everything then